lib/init/variables.html

```diff
@@ -111,14 +111,14 @@ <h2>List of selected (GRASS related) shell environment variables</h2>
 
   <dt>GRASS_COMPRESSOR</dt>
   <dd>[libraster]<br>
-    the compression method for new raster maps can be set with the
-    environment variable GRASS_COMPRESSOR. Supported methods are RLE, 
-    ZLIB, LZ4, BZIP2, and ZSTD. The default is ZSTD if available, 
-    otherwise ZLIB, which can be changed with e.g. 
-    <tt>GRASS_COMPRESSOR=ZSTD</tt></dd>, granted that GRASS has been 
-    compiled with the requested compressor. Compressors that are always 
-    available are RLE, ZLIB, and LZ4. The compressors BZIP2 and ZSTD 
-    must be enabled when configuring GRASS for compilation.
+    the compression method for new raster maps can be set with 
+    GRASS_COMPRESSOR as environment variable or as gisenv variable. 
```

Comment:
```suggestion
    GRASS_COMPRESSOR as environment variable or as <b>g.gisenv</b> defined variable. 
```

Just a suggestion, as a pure "gisenv" may no be clear...